test/fixtures/networks.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "projects/modular-ethos-98809/global/networks",
```

Just a nitpick and I know this is not developer friendly, but it would be nice to keep the same project over the all fixtures.
Can we go with the `coastal-haven-123456`?
```suggestion
  "id": "projects/coastal-haven-123456/global/networks",
```